processing/dwf_calculation_algorithm.py

```diff
@@ -73,10 +73,17 @@
     [22, 0.045],
     [23, 0.04],
 ]
+INFLOW_0D_NONE = 0
+INFLOW_0D_IMPERVIOUS_SURFACE = 1
+INFLOW_0D_SURFACE = 2
+INFLOW_TABLE_NAME_BASES = {
+    INFLOW_0D_NONE: None,
```

Comment:
ik denk dat dit er beter uit kan aangezien dit niet voor kan komen / geen geldige tabelnaam is.

```
INFLOW_TABLE_NAME_BASES = {
    INFLOW_0D_IMPERVIOUS_SURFACE: "impervious_surface",
    INFLOW_0D_SURFACE: "surface",
}
```

Reply:
zojuist aangepast en gepusht